src/app/_components/NetworkOverview/TimeframeSelector.tsx

```diff
@@ -0,0 +1,37 @@
+import { MenuContent, MenuItem, MenuRoot, MenuTrigger } from '@/components/ui/menu';
```

Comment:
This is going to be used throughout the app, so can we move this to the components folder

Reply:
That would mean also making this component work for other components as well